### PR TITLE
Replacing glog.Errof() with util.HandleError()

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Changed `glog.Errorf(<>)` into `util.HandleError(fmt.Errorf(<>))` and `glog.Error(<>)` into `util.HandleError(<>)` in code outside of `origin/pkg/cmd` where the error was eaten.

Works on #2628 @smarterclayton

### CODE REVIEW

**Comment by ncdc** [comment]
I saw the description for this PR and thought you were trying to switch to a different logger, ha.

**Comment by stevekuznetsov** [comment]
@ncdc made that clearer!

**Comment by stevekuznetsov** [comment]
[test]

File: pkg/assets/handlers.go [hunk 1]
```diff
@@ -186,7 +186,7 @@ func GeneratedConfigHandler(config WebConsoleConfig, h http.Handler) http.Handle
 			w.Header().Add("Cache-Control", "no-cache, no-store")
 			w.Header().Add("Content-Type", "application/json")
 			if err := configTemplate.Execute(w, config); err != nil {
-				glog.Errorf("Unable to render config template: %v", err)
+				util.HandleError(fmt.Errorf("Unable to render config template: %v", err))
```
Comment (kargakis) [inline pkg/assets/handlers.go]: Mind updating all those error message created with `fmt.Errorf` start with a lowercased letter?

Comment (stevekuznetsov) [inline pkg/assets/handlers.go]: It can be done

**Comment by stevekuznetsov** [comment]
@kargakis  changed the capitalization conventions - anything else you see missing?

**Comment by kargakis** [comment]
LGTM

**Comment by smarterclayton** [comment]
[merge]

----- Original Message -----

**Pull request merged**